source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
@vitkarpov As far as I get, you suggest to replace `window` with a local object (like, create `storage.js` with content like `export const global = {}`).

Am I right? If yes, this doesn’t look good to me. Most likely, `window` is currently used to share the data between different instances of Likely that are used on the same page. (I’m not sure though because that was written before me.) Therefore, removing can cause hardly noticeable side effects. I don’t think I have time to do real manual testing around this at the moment, so I’d leave this change as is.

(As a side point: if everything is OK and `window` isn’t necessary there, `const updateCount = {}` would be even more enough then. (That is, without `storage.js`.))